R/sensitivity-calculation.R

```diff
@@ -16,11 +16,17 @@
 #'   the following parameters will be considered: `"C_max"`, `"t_max"`,
 #'   `"AUC_inf"`. If `NULL`, all available PK-parameters will be calculated. You
 #'   can also specify custom PK parameters.
+#' @param customOutputFunctions A named list with
+#' custom function(s) for PK parameter calculation. User-defined functions should
+#' have either 'x', 'y', or both 'x' and 'y' as parameters which correspond to
+#' x-Dimension (time) or y-Dimension values from simulation results.
+#' @param ... Additional parameters passed to the function.
```

Comment:
The ellipsis (`...`) is part of the `customOutputFunctions` signature? The way it is written here, it is part of the `sensitivityCalculation`  function, which is not correct. Am I right?